Design.md

```diff
@@ -0,0 +1,96 @@
+
+These 1x1 degree COGs are then summarized into a set of files that are ingestable into a [MapBox tileset source](https://docs.mapbox.com/mapbox-tiling-service/guides/tileset-sources/), the core unit of the MapBox Tiling Service. The requirements for this are as follows:
+- UInt8 datatype
+- Total dataset size of less than 50 GB
+- No file greater than 10 GB
+- No more than 10 input files per tileset source
+A single tile map layer can be composed of multiple tileset sources if necessary to get around the 10 file limit, but testing has shown that we can meet the requirements above for a single tileset source that includes all of North America, which is sufficient for this project.
+
+To go from the 1x1 degree COGs to a MapBox tileset source we must mosaic the 1x1 degree COGs into 10 or fewer tiled GeoTiffs with a UInt8 datatype. GeoTiffs are used because the overviews included in COGs are unused by MapBox, and only serve to increase file size. Dataset-specific discussions will need to had with the OPERA team concerning how to represent the datasets as UInt16 datatypes, but an initial guess of `round(pixel_value,2)*100) + 1` will be used for all datsets during initial development. Values above and below this range will be clipped to 255 and 0 respectively.
+
+LZW or another compression method will be used for all files to reduce file sizes.
+
+## Cumulative Displacement
+### Format
+To fully understand the context of InSAR-derived displacement data (such as the OPERA-Disp products) multiple pieces are required for each pixel:
+
+1. The geographic location of the pixel (from the GDAL metadata)
+2. The start and end date for the displacement observation
+3. The reference pixel location to which all displacement measurements in a product are relative to
+
+This is a lot of information, and we will need to ensure that we provide access to all of this data (or at least make it trackable) for every pixel in the mosaic.
+
+To do this, the 1x1 degree cumulative displacement COGs will have the following components:
+1. A Float32 short wavelength cumulative displacement band sourced directly from the products
+2. A sidecar metadata file (likely a STAC item) containing frame, spatiotemporal reference data, and a link to an external tile frame image (see below)
```

Comment:
> A sidecar metadata file (likely a STAC item)

I'd recommend saving these attributes as key/value metadata attributes of the GeoTIFF itself, rather than in a sidecar file.